config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
+
 Style/Documentation:
   Enabled: false
 
+Style/EmptyLinesAroundBlockBody:
+  Enabled: false
+
 Style/EmptyLinesAroundClassBody:
   Enabled: false
 
 Style/EmptyLinesAroundModuleBody:
   Enabled: false
 
+Style/ExtraSpacing:
+  Enabled: false
```

Comment:
I'm not thrilled wrt to the end of line comments -- I think having them mushed against the code is bad.  Perhaps minimizing the need for such comments, or putting them above the line when we do need them, would assuage my concern.   

I'm also not thrilled at mucking about with line breaks and indentation that doesn't make a diff to code readability.  But, as I said, I *think* turning some of the other alignment rules off will address much of this.

With those two caveats stated, I can live with this rule.